package.json

```
@@ -17,7 +17,7 @@
     "storybook": "NODE_ENV=development start-storybook -p 8180 -c .storybook",
     "test": "npm run test:lint && npm run test:lint:css && npm run test:unit && npm run test:packages",
     "test:packages": "lerna run test",
-    "test:lint": "eslint --ext .js,jsx,json ./packages ./scripts",
+    "test:lint": "eslint --ext .js,jsx,json ./packages ./scripts --fix",
```

Are we certain we want the `--fix` here? I'm not convinced everyone will be happy (or no confused) why the code they had saved locally then showed up different in github because the linter autofixed their code on commit because `husky` runs `test:lint` as part of the `pre-commit` hook

---

I would rather be told there are linting problems and fix it how I like locally. That might be using `--fix` or that might be me manually fixing it.

So I'd be in favour of removing `--fix`

---

imo we shouldnt, --fix will fix issues on CI, leaving them wrong on latest. If we want a command to do --fix, it should be seperate to the one CI uses to vet styling IMO

---

Think that confirms it, let's not `--fix` 

```suggestion
    "test:lint": "eslint --ext .js,jsx,json ./packages ./scripts",
```